pkg/local_object_storage/shard/status.go

```diff
@@ -0,0 +1,33 @@
+package shard
+
+import (
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/blobstor"
+	meta "github.com/nspcc-dev/neofs-node/pkg/local_object_storage/metabase"
+	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/writecache"
+	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
+)
+
+type ObjectStatus struct {
+	Blob       blobstor.ObjectStatus
+	Metabase   meta.ObjectStatus
+	Writecache writecache.ObjectStatus
+	Errors     []error
```

Comment:
It's really unobvious what this slice means without looking into the implementation.